tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py

```diff
@@ -698,36 +694,35 @@ def test_in_subgraph_homo():
     # Initialize data.
     total_num_nodes = 5
     total_num_edges = 12
-    indptr = torch.LongTensor([0, 3, 5, 7, 9, 12])
-    indices = torch.LongTensor([0, 1, 4, 2, 3, 0, 1, 1, 2, 0, 3, 4])
+    indptr = torch.tensor([0, 3, 5, 7, 9, 12], device=F.ctx())
+    indices = torch.tensor([0, 1, 4, 2, 3, 0, 1, 1, 2, 0, 3, 4], device=F.ctx())
```

Comment:
> @yxy235 Can you help double-check the tests?

LGTM. Maybe we do not need to set device for indptr and indices when constructing csc_sampling_graph.

Reply:
Yeah, I must have blindly moved them to the device.